samples/wordcount-function.yaml

```diff
@@ -9,4 +9,4 @@ spec:
   output: greetings
   env:
   - name: FUNCTION_URI
-    value: /resources/script/wordcount.sh
+    value: http://sample-functions/script/wordcount.sh
```

Comment:
does this work?

Reply:
not sure, probably not, didn't have instructions for how to run this sample